dashboard/test/models/plc/user_course_enrollment_test.rb

```diff
@@ -1,30 +0,0 @@
-require 'test_helper'
```

Comment:
There's no real logic in this controller right now - will bring these tests back in some capacity when we are building the logic to assign units to users enrolled in courses